aip_xx1_gen2_launch/launch/pointcloud_preprocessor.launch.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2020 Tier IV, Inc. All rights reserved.
```

Comment:
```suggestion
# Copyright 2024 Tier IV, Inc. All rights reserved.
```

Reply:
I fixed the copyright description in [d75f529](https://github.com/tier4/aip_launcher/pull/326/commits/d75f52983ad62b35cd70182e4bc6d589ffede84d)